### PR TITLE
getting to green: fixed up unit tests

### PR DESCRIPTION
**Pull request by ryanfkeepers**
Gets all onedrive/sharepoint unit tests working.

---

#### Does this PR need a docs update or release note?

- [x] :no_entry: No

#### Type of change

- [x] :broom: Tech Debt/Cleanup

#### Issue(s)

* #1996

#### Test Plan

- [x] :zap: Unit test
- [x] :green_heart: E2E

### CODE REVIEW

File: src/.gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 dist/
 corso
+
+test.log
```
Comment (pandeyabs) [inline src/.gitignore:4]: Should we change this to *.log?

File: src/internal/connector/onedrive/collection.go [hunk 2]
```diff
@@ -174,7 +174,7 @@ func (oc *Collection) Add(item models.DriveItemable) bool {
 	_, found := oc.driveItems[ptr.Val(item.GetId())]
 	oc.driveItems[ptr.Val(item.GetId())] = item
 
-	return !found // !found = new
+	return !found // if !found, it's a new addition
```
Comment (pandeyabs) [inline src/internal/connector/onedrive/collection.go:177]: nit: move comment above the line of code.

File: src/internal/connector/onedrive/item.go [hunk 3]
```diff
@@ -23,63 +22,54 @@ var downloadURLKeys = []string{
...
 
-type augmenterAndGetter interface {
-	ItemInfoAugmenter
-	api.Getter
-}
-
 func downloadItem(
 	ctx context.Context,
-	aag augmenterAndGetter,
+	ag api.Getter,
```
Comment (pandeyabs) [inline src/internal/connector/onedrive/item.go:27]: This is better!

**Review by pandeyabs — APPROVED** [review]

**Comment by pandeyabs** [comment]
Left a few nits. Otherwise, LGTM

File: src/.gitignore (hunk 1 above)
Comment (ryanfkeepers) [reply src/.gitignore:4]: not a bad idea

**Pull request merged**